app/views/directives/notifications/notification-body.html

```diff
@@ -10,6 +9,11 @@
     <span class="sr-only">Clear notification</span>
     <span aria-hidden="true" class="pficon pficon-close"></span>
   </button>
+  <button
```

Comment:
I would only expect this button to exist if the notification is unread

Reply:
Maybe move it down after the notification message, too?

Reply:
Will do.